src/c_peering.cpp

```diff
@@ -43,7 +43,8 @@ void c_peering::print(ostream & ostr) const {
 }
 
 void c_peering::send_data(const char * data, size_t data_size) {
-	pfp_UNUSED(data); pfp_UNUSED(data_size);
+	pfp_UNUSED(data);
+	pfp_UNUSED(data_size);
 	pfp_throw_error( std::runtime_error("Used abstract send_data() that does nothing") );
```

Comment:
WTF? If this method is "abstract" why have a body?

Reply:
for turn off compilation warnings.

Reply:
I think it is a huge design error but not yours.